content/docs/reference/platform-api.md

```diff
@@ -0,0 +1,31 @@
++++
```

Comment:
The details in this and the distribution doc seem a bit sparse compared to the buildpack one, but it's still helpful to have them.

Reply:
I didn't want to copy/paste the spec, so I tried to narrow this down to essentials (and things that wouldn't change frequently.

Reply:
that said, I'm happy to add more if you think we should